apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -1,47 +1,69 @@
 import {DetailViewContents} from '@cdo/apps/code-studio/pd/application_dashboard/detail_view_contents';
 import {ApplicationStatuses, ApplicationFinalStatuses} from '@cdo/apps/code-studio/pd/application_dashboard/constants';
 import React from 'react';
+import _ from 'lodash';
 import {expect} from 'chai';
 import {mount} from 'enzyme';
 
 describe("DetailViewContents", () => {
-  const mountDetailView = (applicationType) => {
+  const mountDetailView = (applicationType, applicationDataOverrides = {}) => {
+    const defaultApplicationData = {
+      regionalPartner: 'partner',
+      notes: 'notes',
+      status: 'accepted',
+      school_name: 'School Name',
+      district_name: 'District Name',
+      email: 'email',
+      application_type: applicationType,
+      form_data: {
+        firstName: 'First Name',
+        lastName: 'Last Name',
+        title: 'Title',
+        phone: 'Phone',
+        preferredFirstName: 'Preferred First Name',
+        accountEmail: 'accountEmail',
+        alternateEmail: 'alternateEmail',
+        program: 'program',
+        planOnTeachering: ['Yes'],
```

Comment:
is this supposed to be 'teachering'?

Reply:
Ha, no but I like it :). This code was just moved - I didn't change it and didn't notice. Good catch. 

Actually this value, only present in facilitator applications, isn't specified in the props nor asserted in this test. I'm just going to remove it.

I accidentally merged, so will remove this in a followup...